lib/vdsm/storage/lvmdevices.py

```diff
@@ -65,8 +66,12 @@ def configure(vgs):
     # On the other hand configuring correct devices file doesn't cause any
     # harm.
```

Comment:
Read the function docstring - looks like we must call `_create_system_devices()` in all cases to ensure that lvm filter is removed, and empty devices file must be created by that function. Please move the fix down.

Reply:
I read the docstring. I think the comment is probably not considering the case where `vgs` is empty, as in that case `_create_system_devices` clearly does not do anything.

Anyway, the filter is removed in `lvmfilter.remove_filter()`, and that part of the code logic hasn't changed. It is still executed if we do not call `_create_system_devices`

Reply:
This is a bug in _create_system_devices(), it did not implement the required behavior expected by the configure function.